pkgs/by-name/ss/sshwifty/package.nix

```
@@ -0,0 +1,61 @@
+{
+  lib,
+  buildGoModule,
+  buildNpmPackage,
+  fetchFromGitHub,
+  go,
+}:
+
+let
+  version = "0.3.25-beta-release";
+
+  sshwifty-ui = buildNpmPackage {
+    pname = "sshwifty-ui";
+    inherit version;
+
+    src = fetchFromGitHub {
+      owner = "nirui";
+      repo = "sshwifty";
+      rev = version;
+      hash = "sha256-gcr/l3fV7s1cwoTGrdMiT1JyxxQOGLrY1qHOBt9oy3w=";
+    };
+
+    npmDepsHash = "sha256-Suo678zynlRqcsY5G0LL4hailRjyi7TWwO5Xs3ACKKI=";
+
+    npmBuildScript = "generate";
+
+    postInstall = ''
+      for i in static_pages static_pages.go; do
+        cp -r application/controller/$i \
+          $out/lib/node_modules/sshwifty-ui/application/controller
+      done
```

```suggestion
      cp -r application/controller/{static_pages,static_pages.go} \
        $out/lib/node_modules/sshwifty-ui/application/controller
```
we could simplify this a bit but on you